e2e/cypress/integration/channel_sidebar/hotkeys_spec.js

```diff
@@ -71,6 +74,9 @@ describe('Channel switching', () => {
         // * Verify that we've switched to the new team
         cy.get('#headerTeamName').should('contain', teamName);
 
+        // # Post any message
+        cy.postMessage('hello');
```

Comment:
Keypress intermittently works. Posting a message is a workaround here to ensure that a page is fully loaded before initiating shortcut keys.